app/views/casework/crime_applications/sections/_housing_payments.html.erb

```diff
@@ -28,7 +28,8 @@
 
     <%= render partial: 'payment_with_frequency', locals: {
           key: label_text("how_much_#{payment.payment_type}"),
-          payment: payment
+          payment: payment,
+          does_not_text: t(:does_not_get, scope: 'values')
```

Comment:
The income benefits and payments screens display 'do not get' but the text is slightly different for outgoing payments 'do not pay'. So i opted to pass in the value to the partial. Is this a good term for the local variable? Does it make sense?